Scrot.py

```diff
@@ -0,0 +1,78 @@
+"""Extension wich basically wraps the commandline utility scrot to make
+screenshots from albert. The extension supports taking screenshots of the whole
+screen, an specific area or the current active window.
+
+When the screenshot was made you will hear a sound wich indicates that the
+screenshot was taken successfully.
+
+Screenshots will be saved in your home directory by default."""
+
+from albertv0 import *
+from shutil import which
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "scrot"
```

Comment:
If the pretty name is only "scrot", people who don't know the scrot utility won't know what this extension is. How about "scrot screenshot utility"?